integration-tests/path-prefix/src/pages/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Link } from 'gatsby'
+
+import Layout from '../components/layout'
+
+const IndexPage = () => (
+  <Layout>
+    <h1>Hi people</h1>
+    <p>Welcome to your new Gatsby site.</p>
+    <p>Now go build something great.</p>
+    <Link data-test-id="page-2-link" to="/page-2/">
```

Comment:
[Because Kent C Dodds](https://blog.kentcdodds.com/making-your-ui-tests-resilient-to-change-d37a6ee37269)